### PR TITLE
fixing issue on Avalanche Icons

### PR DESCRIPTION
**Pull request by alvaro-ricotta**
Simple hard fix for Avalanche icons on Earn page. 

image

### CODE REVIEW

**Comment by alvaro-ricotta** [comment]
@sk-enya not sure what is EVO token, but we need to add this token into our tokenlist.

File: packages/boba/gateway/src/util/coinImage.ts [hunk 1]
```diff
@@ -4,6 +4,12 @@ import mttLogo from 'assets/images/mtt.png'
 export const getCoinImage = (symbol: string): string => {
   const logoURIbase =
     'https://raw.githubusercontent.com/bobanetwork/token-list/main/assets'
+
+  // hard fox for avax
```
Comment (InoMurko) [inline packages/boba/gateway/src/util/coinImage.ts:8]: 🦊

**Comment by sk-enya** [comment]
> @sk-enya not sure what is EVO token, but we need to add this token into our tokenlist.

for your reference @alvaro-ricotta 
https://support.avax.network/en/articles/6349868-what-does-the-e-suffix-in-the-token-name-mean-on-the-avalanche-bridge

**Review by sk-enya — CHANGES_REQUESTED** [review]
It looks confusing to user as you can the two different tokens with same icons.. 
as `.e` has some context and meaning to it like `it's bridge token on avax from ethereum` so better we should image icons like below.
images

[refere link for more details](https://support.avax.network/en/articles/6349868-what-does-the-e-suffix-in-the-token-name-mean-on-the-avalanche-bridge)

let's get some help from @yhong1073 to get the icons prepared.



List of icons which I found of .e type, @yhong1073 can you please help us to prepare icon design to update it on gateway to accordingly?

1. USDT.e
2. USDC.e
3. BUSD.e
4. DAI.e

**Comment by yhong1073** [comment]
[avax icons.zip](https://github.com/bobanetwork/boba/files/12600206/avax.icons.zip)

Here are the icons.

**Comment by alvaro-ricotta** [comment]
Hey @sk-enya sounds good in that case, we can remove this PR and just update the tokenlist github, also @yhong1073 we need a icon for EVO token, not sure what token it´s , @InoMurko have some idea which token we´re talking about EVO?

**Comment by yhong1073** [comment]
evo
Here is the image for EVO. The image quality is bad because there were only low quality images for this logo.

**Comment by sk-enya** [comment]
@alvaro-ricotta , I guess you are good to close this and update the token list repo itself

**Comment by alvaro-ricotta** [comment]
@yhong1073 tokenlist only allow svg files, can you send a new version only with svgs? once i get it i´ll update our tokenlist repository

**Comment by yhong1073** [comment]
[svg icons.zip](https://github.com/bobanetwork/boba/files/12611047/svg.icons.zip)
Here are the svg versions of the icons.

**Comment by alvaro-ricotta** [comment]
@sk-enya https://github.com/bobanetwork/token-list/pull/12 pr has been created, once we merge it we can close this pr.

**Comment by sk-enya** [comment]
@alvaro-ricotta  https://github.com/bobanetwork/token-list/pull/12 has been merged!!

**Comment by alvaro-ricotta** [comment]
This pr has been closed and remplaced by this branch https://github.com/bobanetwork/token-list/tree/feat/BobaLogov2

**Pull request closed without merging**